chef-config/chef-config.gemspec

```diff
@@ -8,12 +8,20 @@ Gem::Specification.new do |spec|
   spec.authors       = ["Adam Jacob"]
   spec.email         = ["adam@chef.io"]
 
-  spec.summary       = %q{Chef's default configuration and config loading}
+  spec.summary       = %q{Chef Infra's default configuration and config loading library}
   spec.homepage      = "https://github.com/chef/chef"
   spec.license       = "Apache-2.0"
 
   spec.required_ruby_version = ">= 2.6.0"
 
+  spec.metadata = {
+    "bug_tracker_uri" => "https://github.com/chef/chef/issues",
+    "changelog_uri" => "https://github.com/chef/chef/CHANGELOG.md",
```

Comment:
```suggestion
    "changelog_uri" => "https://github.com/chef/chef/blob/master/CHANGELOG.md",
```

This 404s.

Reply:
That was a copy/paste from chef-utils. I fixed it there as well.